package.json

```diff
@@ -80,19 +80,28 @@
         "command": "prismatic.integrations.import",
         "title": "Import Integration",
         "category": "Prismatic",
-        "enablement": "prismatic.workspaceEnabled"
+        "enablement": "prismatic.workspaceEnabled",
+        "icon": "$(cloud-upload)"
       },
       {
         "command": "prismatic.integrations.test",
         "title": "Test Integration",
         "category": "Prismatic",
-        "enablement": "prismatic.testCommandEnabled && prismatic.workspaceEnabled"
+        "enablement": "prismatic.testCommandEnabled && prismatic.workspaceEnabled",
+        "icon": "$(play)"
       },
       {
         "command": "prismatic.prismaticUrl",
         "title": "Prismatic URL",
         "category": "Prismatic",
         "enablement": "prismatic.workspaceEnabled"
+      },
+      {
+        "command": "prismatic.executionResults.refetch",
```

Comment:
[nit] Should we replace all instances of the word "refetch" in the commands/state machines with "refresh"?